modules/apps/depot/depot-test/src/testFunctional/tests/MyDMTest.testcase

```diff
@@ -0,0 +1,76 @@
+@component-name = "portal-document-management"
+definition {
+
+	property testray.main.component.name = "Document Management";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		task ("Given DM Portlet is added to the page") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "DM Henrique Page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Guest",
+				layoutName = "DM Henrique Page",
+				widgetName = "Documents and Media");
+
+			Navigator.gotoPage(pageName = "DM Henrique Page");
+
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Documents and Media");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			DMDocument.tearDownCP();
+
+			JSONLayout.deletePublicLayout(
+				groupName = "Guest",
+				layoutName = "DM Henrique Page");
+
+			User.logoutPG();
+		}
+	}
+
+	@description = "Can Edit a Documents and Media"
+	@priority = 5
```

Comment:
Similar line of thinking as https://github.com/lucasperj/liferay-portal/pull/373/files#r1322156775, but in that case, the scenario have been defined and it is identified as high(FP4), it indicates that one really important function of the feature(DOWNLOAD) in question should't brake, but that action it's not related to CRUD (Create - Read - Update - Delete), not being an operation that stopped the end user's operations if it broke, but still of great importance that could greatly degrade the usability of the product.

If you have questions or want to discuss the topic further, you can send me a message on slack